src/ripple/protocol/TER.h

```diff
@@ -298,7 +299,8 @@ enum TECcodes : TERUnderlyingType {
     tecUNFUNDED_AMM = 162,
     tecAMM_BALANCE = 163,
     tecAMM_FAILED = 164,
-    tecAMM_INVALID_TOKENS = 165
+    tecAMM_INVALID_TOKENS = 165,
+    tecEMPTY_DID = 166
```

Comment:
This conflicts with PR #4626: see [lines 302-305 of TER.h](https://github.com/XRPLF/rippled/pull/4626/files#diff-570760ccc79d416a7e98bf3e851c2e298ea5cf90f7207018e06aa6b46d09a013R302-R305):

```
    tecAMM_EMPTY = 166,
    tecAMM_NOT_EMPTY = 167,
    tecAMM_ACCOUNT = 168,
    tecINCOMPLETE = 169
```

Reply:
I'll fix that after I rebase when that PR is merged - I don't think it's a super urgent fix.

Reply:
This has been resolved (I don't remember which commit)